cores/piduino/WString.h

```diff
@@ -123,5 +123,8 @@ class StringSumHelper : public String
   StringSumHelper(double num) : String(num) {}
 };
 
+typedef char __FlashStringHelper;
+#define PROGMEM
+
```

Comment:
add pgmspace.h in the core folder and I'll make it so it can be included as avr/pgmspace.h as well